gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -252,14 +252,20 @@ func (p *glusterBlockProvisioner) createSecret(nameSpace string, secretName stri
 		},
 		Type: chapType,
 	}
-	_, err = p.client.Core().Secrets(nameSpace).Create(secret)
-	if err != nil {
-		return nil, fmt.Errorf("glusterblock: failed to create secret, error %v", err)
-	}
+
 	secretRef := &v1.LocalObjectReference{}
-	if secretRef != nil {
-		secretRef.Name = secretName
-		glog.V(1).Infof("glusterblock: secret [%v]: secretRef [%v]", secret, secretRef)
+	if secret != nil {
```

Comment:
i think this and if secretRef != nil are unnecessary but we can leave it as is. lgtm.